src/helpers.js

```diff
@@ -49,12 +49,15 @@ function getReplacement({code, fileOpts, args = []}, babel) {
 }
 
 function applyReplacementToPath(replacement, path) {
-  if (!replacement) {
```

Comment:
Had to invert the condition because of https://eslint.org/docs/rules/no-negated-condition